extra-network/libbpf/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=libbpf
+PKGSEC=net
+PKGDEP="glibc elfutils linux+api"
+BUILDDEP="pkg-config"
+PKGDES="Automated upstream mirror for libbpf stand-alone build."
```

Comment:
Seriously? Don't just copy GitHub description, it's easy to do, but rarely correct or useful.